### PR TITLE
add signature to NetcdfDataset

### PR DESCRIPTION
**Pull request by lesserwhirls**

### CODE REVIEW

**Comment by lesserwhirls** [comment]
Sorry about the odd diff. The javadoc is same as an older signature, which probably screwed the diff up. Basically creates a new signature for `NetcdfDataset.acquireDataset` that uses the default `enhanceMode`.

**Comment by lesserwhirls** [comment]
Ok, diff is better now...not perfect, but much better.

File: cdm/src/main/java/ucar/nc2/dataset/NetcdfDataset.java [hunk 1]
```diff
@@ -463,6 +463,21 @@ static private CoordSysBuilderIF enhance(NetcdfDataset ds, Set<Enhance> mode, Ca
   }
 
   /**
+   * Same as openDataset, but file is acquired through the File Cache, with defaultEnhanceMode,
+   * without the need of settinging the enhanceMode via the signature.
```
Comment (cwardgar) [inline cdm/src/main/java/ucar/nc2/dataset/NetcdfDataset.java]: Typo.

**Comment by cwardgar** [comment]
Nevermind, fixed it myself in the GitHub UI. Slick!

**Pull request merged**